src/app/main/component/ubs/components/ubs-main-page/ubs-main-page.component.ts

```diff
@@ -61,4 +62,10 @@ export class UbsMainPageComponent {
     'ubs-homepage.ubs-courier.rules.content.li_2',
     'ubs-homepage.ubs-courier.rules.content.li_3'
   ];
+
+  constructor(private router: Router) {}
+
+  redirectToOrder() {
+    this.router.navigate(['ubs/order']);
```

Comment:
Isn't it supposed to be ['ubs', 'order']? I guess this way is more verbose.

Reply:
fixed